compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/Types.java

```diff
@@ -2200,6 +2201,13 @@ private boolean isInSameVisibilityRegion(BSymbol lhsSym, BSymbol rhsSym) {
         return !Symbols.isPrivate(rhsSym) && !Symbols.isPublic(rhsSym) && lhsSym.pkgID.equals(rhsSym.pkgID);
     }
 
+    private boolean isSameMethodType(BSymbol lhsSym, BSymbol rhsSym) {
```

Comment:
Same comment as for the jvm/TypeChecker.

Reply:
Changed